Command/RunCommand.php

```diff
@@ -140,13 +140,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         while (time() - $startTime < $maxRuntime) {
             $this->checkRunningJobs();
-
-            $excludedIds = array();
+            
             while (count($this->runningJobs) < $maxConcurrentJobs) {
                 try {
                     $this->preFindStartableJob();
                     $this->getRepository()->startTrasaction();
-                    if (null === $pendingJob = $this->getRepository()->findStartableJob($queueName, $excludedIds)) {
+                    if (null === $pendingJob = $this->getRepository()->findStartableJob($queueName)) {
```

Comment:
Talked with @amit-revinate. Will move it to test env first. And see whether it is causing any fatal issues.